Jenkinsfile

```diff
@@ -13,7 +13,7 @@ commit = ''
 
 ircMsgResult(CHANNELS) {
     ystage('Test') {
-        node {
+        ynode.forConfiguredHostType(ownerName: 'Yelp', repoName: 'paasta') {
```

Comment:
is this going to do the right thing internally? there's no Yelp org in our GHE

Reply:
it won't from what I see. basically this is used to check whether owner/repo name is in a mesos-opt-in list. In this case, we can just add "Yelp/paasta" to the list if we ever want to use mesos for it

Reply:
should we just use a raw ynode? there's also no jenkins.yaml here so there's no host type to automatically pick up

Reply:
```suggestion
        ynode {
```